cypress/tests/ui/auth.spec.ts

```diff
@@ -1,23 +1,99 @@
 describe("User Sign-up and Login", function () {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/")
+  })
+  
   it("should redirect unauthenticated user to signin page", function () {
+    cy.location("pathname").should("equal", "/signin")
   });
 
   it("should redirect to the home page after login", function () {
+    cy.get("#username").should("exist").type("Katharina_Bernier")
+    cy.get("#password").should("exist").type("s3cret")
+    cy.get("[data-test=signin-submit]").should("exist").click()
+    cy.location("pathname").should("equal", "/")
   });
 
   it("should remember a user for 30 days after login", function () {
-    // check cookie `connect.sid` and its property `expiry` 
-  });
+    // check cookie `connect.sid` and its property `expiry`
+    const date = new Date()
+    const plus30Days = date.setDate(date.getDate() + 30)
+    const currentTimestamp = Math.floor(plus30Days / 1000)
+    const currentTimeInSeconds = Math.floor(currentTimestamp / 10)
```

Comment:
This could be moved somewhere to utilities (wrap this code into function and put to `cypress/support/utils.ts`)